v3-sdk/collecting-fees/src/libs/liquidity.ts

```diff
@@ -67,31 +77,62 @@ export async function collectFees(
   return sendTransaction(transaction)
 }
 
-export async function getPositionIds(
-  provider: ethers.providers.Provider,
-  address: string,
-  contractAddress: string
-): Promise<number[]> {
-  // Get currency otherwise
+export async function getPositionIds(): Promise<number[]> {
+  const provider = getProvider()
+  if (!provider) {
+    throw new Error('No provider available')
```

Comment:
Is it intentional to return vs not throw here?

Reply:
Yep, a missing provider should throw an error!